### PR TITLE
Remove cookie badge when a link is available

### PR DESCRIPTION
**Pull request by chrisguindon**
Signed-off-by: Christopher Guindon <chris.guindon@eclipse-foundation.org>

### CODE REVIEW

File: js/solstice.cookieconsent.js [hunk 1]
```diff
@@ -70,8 +70,7 @@ window.addEventListener("load", function() {
       }
     },
     onInitialise: function(status, options) {
-      var toolbar = document.getElementsByClassName("toolbar-manage-cookies");
-      if (toolbar.length <= 0) {
+      if (!document.getElementsByClassName("toolbar-manage-cookies").length) {
```
Comment (ericpoirier) [inline js/solstice.cookieconsent.js:73]: I tested this locally and it does not work as is.

By doing this, it works:

```
if (!document.getElementsByClassName("toolbar-manage-cookies").length) {
  setTimeout(function() {
    document.getElementsByClassName("cc-revoke")[0].style.display = "block";
  });
}else {
  document.getElementsByClassName("cc-revoke")[0].style.display = "none";
}
```

**Review by ericpoirier — CHANGES_REQUESTED** [review]

File: js/solstice.cookieconsent.js (hunk 1 above)
Comment (chrisguindon) [reply js/solstice.cookieconsent.js:73]: Thanks

**Review by ericpoirier — APPROVED** [review]

**Comment by chrisguindon** [comment]
Not working... Now the issue is that the banner does not open.

**Review by ericpoirier — APPROVED** [review]

**Comment by chrisguindon** [comment]
This latest update now works for me.

**Pull request merged**